Examples/Framework/src/EventData/ScalingCalibrator.cpp

```diff
@@ -0,0 +1,148 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <Acts/Definitions/TrackParametrization.hpp>
+#include <ActsExamples/EventData/ScalingCalibrator.hpp>
+
+#include <regex>
+
+#include <TKey.h>
+
+namespace detail {
+
+std::pair<Acts::GeometryIdentifier, std::string> parseMapKey(
+    const std::string& mapkey) {
```

Comment:
This is needed because you encode the geo id in the ROOT histogram name as a string, correct?

Reply:
Yes, that's the idea. Probably not the cleanest way to do it...